README.md

```diff
@@ -135,21 +135,46 @@ logging_inputs:
     type: basics
 ```
 
-3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the local system and remote rsyslog and outputs into the local files.
+3) Deploying basic LSR/Logging config files in /etc/rsyslog.d, which handle inputs from the remote rsyslog and outputs into the per host base local files.
 ```
-logging_enabled: true
-rsyslog_capabilities: [ 'network', 'remote-files' ]
-logging_purge_confs: true
 logging_outputs:
-  - name: local-files
-    type: files
+  - name: remote_files_output
+    type: remote_files
 logging_inputs:
-  - name: system-and-remote-input
-    type: basics
+  - name: remote_udp_input
+    type: remote
+    udp_port: 11514
+  - name: remote_tcp_input
+    type: remote
+    tcp_port: 22514
 logging_flows:
-  - name: flow0
-    inputs: [system-and-remote-input]
-    outputs: [local-files]
+  - name: flow_0
+    inputs: [remote_udp_input, remote_tcp_input]
+    outputs: [remote_files_output]
+```
+If more detailed outputs to be configured instead of using the default paths, they are configurable as follows.
+```
+logging_outputs:
+  - name: remote_files_output0
+    type: remote_files
+    remote_log_path: /var/log/remote/%HOSTNAME%/%PROGRAMNAME:::secpath-replace%.log
+    severity: info
+    exclude: ;authpriv.none
```

Comment:
this will change if https://github.com/linux-system-roles/logging/pull/114 merges first

Reply:
Right. I'll fix the conflicts...